app/templates/task/statistics.hbs

```diff
@@ -1,4 +1,5 @@
 <h3 class="visible-print">Statistika</h3>
+<div style="margin: 20px;">

```

Comment:
Po konzultaci s Aničkou tento margin zruším, přijde mi tam stejně navíc jako všechny ty, které tímhle pull requestem odděláváš.

Reply:
Nesouhlasím. Teď to nesedí s okraji ve všech ostatních tabech u úlohy. Jsem pro tam `margin: 20px;` vrátit, ať je to stejné ve všech tabech, které se týkají úlohy. (Zadání, Statistika, Řešení, ...)

Reply:
Žádný jiný tab nemá tak výrazný barevný obsah skrz celou šířku, takže to nikde jinde není tak vidět.
Nejblíž je tomu Hodnocení a jeho sekce Výsledky, takže pokud chceme synchronizovat, jsem pro to to dělat pouze v tabech s podobným obsahem.
Když se do Statistika přidá ten margin, je to viditelně zaříznuté a dle mého názoru to spíš vypadá jako chyba zobrazení, než designový záměr. :/
Ale pokud na tom trváš, určitě to přidat můžeme, v důsledku je to fakt jen drobnost.

Reply:
Hmm, dokud bylo Opravování plné mezer, taky to nikomu nepřišlo divné, to je podle mě jen o zvyku. Zeptám se jinak: proč by měla mít Statistika jiný  margin než všechny ostatní sekce? Já osobně nevidím důvod.

Reply:
Text by měl mít okraje, grafika může být klidně skrz celou šířku.

Reply:
Ok, nejsem grafik, takže tomuto argumentu bohužel nerozumím. Můj názor je ten, že by to mělo být všude stejně, avšak je to celkem prkotina, takže na tom netrvám. Rozhodni se podle vlastního uvážení. Obesílat nějak hromadněji orgy se žádostí o vyjádření mi přijde v současné situaci jako zbytečný overkill.

Reply:
díky :)